src/spasht/ui/template/agent_spasht_ui_content.html.twig

```diff
@@ -0,0 +1,34 @@
+{# Copyright 2019
+   Author: Vivek Kumar<vvksindia@gmail.com>
+   
+   Copying and distribution of this file, with or without modification,
+   are permitted in any medium without royalty provided the copyright notice and this notice are preserved.
+   This file is offered as-is, without any warranty.
+#}
+
+<div class="row" id="spashtStart">
+  <div class="col" ></div>
+  <div class="col-md-4" id="searchBoxCover">
+    {% include 'agent_spasht_search.html.twig' %}
+  </div>
+  <div class="col">
+  </div>
+</div>
+<div class="row" id="spashtStart">
+  <div class="col">
+    {% if advanceSearch == "checked" %}
+      {% include 'agent_spasht_advanceSearch_form.html.twig' %}
+    {% elseif pageNo == 2 %}

```

Comment:
Why not use Strings or constants to identify pages. That would be easier to read and easier to understand.